model/Cargo.toml

```
@@ -11,6 +11,9 @@ primitive-types = { version = "0.7" }
 secp256k1 = "0.17"
 serde = { version = "1.0", features = ["derive"] }
 web3 = { version = "0.13", default-features = false }
+lazy_static = "1.4.0"
+ethabi = "12.0.0"
```

Keeping the version number consistent with the rest of the file
```suggestion
lazy_static = "1.4"
ethabi = "12.0"
```